deploy/kube-config/influxdb/grafana.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         env:
         - name: INFLUXDB_HOST
           value: monitoring-influxdb
-        - name: GRAFANA_PORT
+        - name: GF_SERVER_HTTP_PORT
```

Comment:
[In `grafana/run.sh`, we have transported `GRAFANA_PORT`  to `GF_SERVER_HTTP_PORT`](https://github.com/kubernetes/heapster/blob/master/grafana/run.sh#L6).